interface/drugs/add_edit_drug.php

```diff
@@ -14,6 +14,8 @@
  require_once("drugs.inc.php");
  require_once("$srcdir/options.inc.php");
 
```

Comment:
better to do the following (place this line below the require_once):
use OpenEMR\Services\DrugServices;

Reply:
disregard above comment for now. I am in process of converting this namespace, so can do this in the future.